src/lib.rs

```diff
@@ -565,6 +565,61 @@ pub fn compute_all(
     }
 }
 
+extern "C" {
+    /// Calculates the IV based on www.jaeckel.org. 
+    /// 
+    /// Price is the adjusted_price of an option which can be calculated as
+    /// `let adjusted_price = ot_price / std::f64::consts::E.powf(-risk_free_rate * maturity);`
+    /// where ot_price is the price of the option.
+    /// 
+    /// `F` is the forward_price of the underlying:
+    /// `let forward_price = underlying_price / (-risk_free_rate * maturity).exp();
+    /// 
+    /// `K` is the strike. 
+    /// `T` is the maturity.
+    /// `q` is 1 for call, -1 for put
+    /// `N` is the number of iterations that should be attempted as maximum before stopping. 
+    pub fn implied_volatility_from_a_transformed_rational_guess_with_limited_iterations(
+        price: f64,
+        F: f64,
+        K: f64,
+        T: f64,
+        q: f64, /* q=±1 */
+        N: i64,
+    ) -> f64;
+}
+
+pub fn iv_jaeckel(
```

Comment:
nitpick but if you wanted to match the existing functions, i could see call_iv_jaeckel and put_iv_jaeckel instead of this flag variable

lotta duplicate code i guess but it seems to match how the library is done so far?

or make the flag an enum, kind of confusing... or bool, is_call, is_put, type, etc.